packages/sqlite/src/lib/Snapshot.ts

```diff
@@ -0,0 +1,27 @@
+import { StreamToken, ITimelineEvent } from "@equinox-js/core"
+import * as Token from "./Token.js"
+import { Format } from "./LibSqlClient.js"
+
+export const snapshotCategory = (original: string) => original + ":snapshot"
+
+export const streamName = (category: string, streamId: string) =>
+  `${snapshotCategory(category)}-${streamId}`
+export type Meta = { streamVersion: string }
+const streamVersion = (evt: ITimelineEvent<Format>) => {
+  const meta = JSON.parse(evt.meta ?? "null") as Meta | null
```

Comment:
does parsing the null literal achieve a type hint very cheaply?

Reply:
Nope, in fact, this whole file should be removed because there's no snapshot mechanism yet

Reply:
But now there's a snapshot mechanism!